inference-engine/thirdparty/clDNN/src/gpu/device_info.cpp

```diff
@@ -68,6 +112,9 @@ device_info_internal::device_info_internal(const cl::Device& device) {
     vendor_id = static_cast<uint32_t>(device.getInfo<CL_DEVICE_VENDOR_ID>());
 
     supports_usm = extensions.find("cl_intel_unified_shared_memory") != std::string::npos;
+
+    supports_optimization_hints = false;
```

Comment:
Why is it always disabled?

Reply:
Currently it is not possible to obtain information about optimization hints being supported from OpenCL, as trying to use them may crash whole process.
I wanted to add this flag for easier experimentation and implementation in future.